nyx/nyx_gui/frontend/gui_info.c

```diff
@@ -790,6 +790,78 @@ static lv_res_t _create_window_fuses_info_status(lv_obj_t *btn)
 
 	switch (display_id)
 	{
+	case PANEL_RR_SUPER5_OLED_V1:
+		strcat(txt_buf, "RR SUPER5 OLED");
+		switch (display_rev)
+		{
+		case 0x01:
+			strcat(txt_buf, "-V1");
+			break;
+		default:
+			strcat(txt_buf, " #FFDD00 Contact me!#");
```

Comment:
May be worth saying `Contact Retro Remake!` or something like that, just for people who buy the console from a reseller or something, I know markets like these are pretty popular on the SX Lite